### ISSUE TITLE
Removing Update button from Sales Module

### ISSUE THREAD

**Issue by bill-goldberg** [jekkos/opensourcepos#1322]
I am facing a problem with the update button in Sales Register. When i have to sell more than 10 items then it takes too much time to update each of the single line in the cart.

I want to remove that functionality. Is there any way to handle this ?
![snapcrab_noname_2017-5-29_12-15-32_no-00](https://cloud.githubusercontent.com/assets/23611136/26539182/ac476f3e-4468-11e7-8ca7-91de4676faa4.png)

**Comment by jekkos** [comment]
I think that most of the fields should already auto update after change?
I guess you're then talking about bulk update eg adding a discount and apply it to all items at once? It can be done with a bit of js code

**Comment by bill-goldberg** [comment]
I know that there is a jquery focusout function which automatically reload the page and update the quantity or any input field ... but i also don't want to reload page every time ...

**Comment by daN4cat** [comment]
The right job would be to use a ajax to upload just the updated cell. But that's part of the register rewrite story....

**Comment by SteveIreland** [comment]
I added an ajax call to update temporary item information. That doesn't make any changes to costs or quantities so it wasn't all that hard once I figured out what I needed to do.

If I can find time would anyone have a problem with me adding an ajax update for the quantity, price, and discount (knowing full well that there is a pending rewrite in the queue)?

**Comment by WebShells** [comment]
Mmmm It will be helpful...

**Comment by daN4cat** [comment]
That would be great. Not sure we will ever find the time for a complete rewrite so improving the UI step by step could be a good alternative option.
To be honest I don't like the complete UI refreshing story of the sale register. It's old fashioned way. But I don't have time at the moment to rework it.

**Comment by WebShells** [comment]
@daN4cat i agree, although it takes couple of seconds or ms, but somehow its annoying... Going forward step by step will help since the whole change needs alot of updates and codes to be reworked...
